css/fares.css

```diff
@@ -60,18 +111,30 @@ ul{
         padding-left: 20rem;
         padding-right: 20rem;
     }
+    .header-banner {
+        padding-left: 15rem;
+    }
 }
 @media only screen and (max-width: 1600px) {
     .blue-box{
         padding-left: 10rem;
         padding-right: 10rem;
     }
+    .header-banner {
+        padding-left: 12rem;
+    }
 }
 @media only screen and (max-width: 1400px) {
     .blue-box{
         padding-left: 0;
         padding-right: 0;
     }
+    .header-banner {
+        padding-left: 10rem;
```

Comment:
After adjusting based off comment above, you can delete these lines manually updating the padding